### ISSUE TITLE
Takes 10 seconds to display a message already on my phone

### ISSUE THREAD

**Issue by stoecker** [k9mail/k-9#3003]
After the update for many messages it takes 10 seconds until the text of the message appears.

I click on the message and the header lines with date and folder name appear. Only after 10 seconds the actual text block is shown.

Switching from landscape to portrait mode or back clears screen again and it takes 10 seconds anew until I can see the text.

Samsung A310F Android 7.0, Kernel 3.10.61-11350514

**Comment by jimimaseye** [comment]
I have a similar experience but with non0-downloaded messages too. The message LIST shows the sender and subject line etc (as it did before).  But then tapping the message just shows a blank email which sometimes takes ages (10 seconds?) to display the message, sometimes it simply doesnt get there at all (staying blank) requiring the 'DOWNLOAD MESSAGE' button to be pressed which then shows the message, and sometimes it simply just shows the message immediately (according to the settings - mine is set at 2kb).  Note that this can be with the SAME message and even if it has already previously been viewed/cached completely (it isnt a message size issue or connection issue - proven as this also happens across a LAN connection)

Completely random with no obvious cause.  But this problem do not happen on 5.208.

(Ive learnt to save old APK's of K9 before updating due to recent experiences of it changing/breaking things from what was otherwise a perfectly acceptable client solution).

**Comment by madduck** [comment]
Hello @stoecker, could you please elaborate as to why you closed this issue?

**Comment by stoecker** [comment]
Because it no longer is an issue.

**Comment by madduck** [comment]
That's nice, @stoecker. What fixed it? Because the issue certainly still exists for me. Have you asked @jimimaseye?

**Comment by stoecker** [comment]
One of the updates inbetween. Didn't have this issue in the last months although nothing else changed, so I consider it fixed. If you still have this problem you should either reopen the ticket and supply additional information or open a new ticket.

**Comment by madduck** [comment]
I cannot reopen the ticket. And I don't have additional information. You are the one with the additional information, unfortunately.
